### PR TITLE
Pr review fixes

### PR DESCRIPTION
**Pull request by kavinda1995**
### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/organizations/components/organization-list.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```
Comment (thivi) [inline apps/console/src/features/organizations/components/organization-list.tsx:2]:
```suggestion
 * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

File: apps/console/src/features/identity-providers/data/identity-provider-templates/templates/organization-enterprise-identity-provider/create-wizard-help.tsx [hunk 2]
```diff
@@ -41,21 +42,31 @@ const OrganizationEnterpriseIdentityProviderCreateWizardHelp:
...
                 <Divider/>
 
                 <div data-testid={ testId }>
-                    <Heading as="h5">Description</Heading>
-                    <p>Provide a description for the enterprise authentication provider to explain more about it.</p>
-                    <p>E.g., This is the authenticator for MyOrg, which acts as the IDP for MyApp.</p>
+                    <Heading as="h5">{
+                        t("console:develop.features.authenticationProvider.templates.organizationIDP.wizardHelp" +
+                            ".description.heading")
+                    }</Heading>
```
Comment (thivi) [inline apps/console/src/features/identity-providers/data/identity-provider-templates/templates/organization-enterprise-identity-provider/create-wizard-help.tsx:65]:
```suggestion
                    <Heading as="h5">
                        { t("console:develop.features.authenticationProvider.templates.organizationIDP.wizardHelp" +
                            ".description. Heading") }
                    </Heading>
```

File: apps/console/src/features/identity-providers/data/identity-provider-templates/templates/organization-enterprise-identity-provider/create-wizard-help.tsx [hunk 3]
```diff
@@ -41,21 +42,31 @@ const OrganizationEnterpriseIdentityProviderCreateWizardHelp:
...
+                    <p>{ t("console:develop.features.authenticationProvider.templates.organizationIDP.wizardHelp" +
+                        ".description.description") }</p>
```
Comment (thivi) [inline apps/console/src/features/identity-providers/data/identity-provider-templates/templates/organization-enterprise-identity-provider/create-wizard-help.tsx:67]:
```suggestion
                    <p>
                        { t("console:develop.features.authenticationProvider.templates.organizationIDP.wizardHelp" +
                        ".description.description") }
                    </p>
```

Comment (thivi) [reply apps/console/src/features/identity-providers/data/identity-provider-templates/templates/organization-enterprise-identity-provider/create-wizard-help.tsx:67]: Check the other places as well.

File: apps/console/src/features/identity-providers/data/identity-provider-templates/templates/organization-enterprise-identity-provider/create-wizard-help.tsx (hunk 2 above)
Comment (thivi) [reply apps/console/src/features/identity-providers/data/identity-provider-templates/templates/organization-enterprise-identity-provider/create-wizard-help.tsx:65]: Check the other places as well.

**Review by thivi — DISMISSED** [review]

File: apps/console/src/features/identity-providers/data/identity-provider-templates/templates/organization-enterprise-identity-provider/create-wizard-help.tsx (hunk 3 above)
Comment (kavinda1995) [reply apps/console/src/features/identity-providers/data/identity-provider-templates/templates/organization-enterprise-identity-provider/create-wizard-help.tsx]: Fixed in https://github.com/wso2/identity-apps/pull/3617/commits/0d2293a9f5ef88031f68e9c05ffb4847f410fef8

File: apps/console/src/features/identity-providers/data/identity-provider-templates/templates/organization-enterprise-identity-provider/create-wizard-help.tsx (hunk 2 above)
Comment (kavinda1995) [reply apps/console/src/features/identity-providers/data/identity-provider-templates/templates/organization-enterprise-identity-provider/create-wizard-help.tsx]: Fixed in {
    "clientID": "h_fR9hMIt5wsEbio5EJ95w9Q15oa",
    "baseUrl": "https://api.asg.io/t/testasgardeo",
    "signInRedirectURL": "https://localhost:3000",
    "signOutRedirectURL": "https://localhost:3000",
    "scope": ["openid","profile"]
}

**Review by NipuniBhagya — APPROVED** [review]

**Pull request merged**